packages/amazonq/src/lsp/client.ts

```diff
@@ -72,6 +74,47 @@ export async function startLanguageServer(
     const clientOptions: LanguageClientOptions = {
         // Register the server for json documents
         documentSelector,
+        middleware: {
+            workspace: {
+                /**
+                 * Convert VSCode settings format to be compatible with flare's configs
+                 */
+                configuration: async (params, token, next) => {
+                    const config = await next(params, token)
+                    if (params.items[0].section === 'aws.q') {
+                        const customization = undefinedIfEmpty(getSelectedCustomization().arn)
+                        /**
+                         * IMPORTANT: This object is parsed by the following code in the language server, **so
+                         * it must match that expected shape**.
+                         * https://github.com/aws/language-servers/blob/1d2ca018f2248106690438b860d40a7ee67ac728/server/aws-lsp-codewhisperer/src/shared/amazonQServiceManager/configurationUtils.ts#L114
+                         */
+                        return [
+                            {
+                                customization,
+                                optOutTelemetry: getOptOutPreference() === 'OPTOUT',
+                                projectContext: {
+                                    // TODO uncomment this when local indexing is in agent chat server manifest
+                                    enableLocalIndexing: false,
```

Comment:
thanks! we should be able to set it to true now

Reply:
Done